docs/current/sailbot_workspace/setup.md

```diff
@@ -171,31 +196,33 @@ git clone https://github.com/UBCSailbot/sailbot_workspace.git
 ## 5. Open Sailbot Workspace in a Dev Container
 
 1. Ensure that Docker is running
-2. Run the `Dev Containers: Reopen in Container` command in the
-   [VS Code command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette){target=_blank}
+2. Run the `Dev Containers: Reopen in Container` command
```

Comment:
Should we say Command Prompt instead of command just to make it clearer?

Reply:
This is a VS Code command, not a terminal command. I will explicitly mention this every time I reference one.

I think tasks and launch configurations are unique to VS Code so I don't need to prefix them with `VS Code`. What do you think?